### ISSUE TITLE
ecppack is crashing on Windows (MSYS2)

### ISSUE THREAD

**Issue by umarcor** [stnolting/neorv32-setups#11]
```
# ecppack
Mingw-w64 runtime failure:
32 bit pseudo relocation at 00007FF784896911 out of range, targeting 00007FFAB975DA10, yielding the value 0000000334EC70FB.
```

That is producing failures in some CI jobs. GHDL, Yosys and nextpnr are successful, bit the bitstream generation crashes when running `ecppack`.

**Comment by umarcor** [comment]
This is now fixed, since msys2/MINGW-packages#10573 was merged and released. See https://github.com/stnolting/neorv32-setups/actions/runs/1726218236

**Comment by stnolting** [comment]
Great! Thanks!!
